Userland/Libraries/LibJS/AST.h

```diff
@@ -320,52 +319,74 @@ class ExportStatement final : public Statement {
 public:
     static FlyString local_name_for_default;
 
+    // ExportEntry Record, https://tc39.es/ecma262/#table-exportentry-records
     struct ExportEntry {
         enum class Kind {
-            ModuleRequest,
-            LocalExport
+            NamedExport,
+            ModuleRequestAll,
+            ModuleRequestAllButDefault,
         } kind;
-        // Can always have
-        FlyString export_name;
 
-        // Only if module request
-        ModuleRequest module_request;
+        FlyString export_name;          // [[ExportName]]
+        FlyString local_or_import_name; // Either [[ImportName]] or [[LocalName]]
+
+        ExportEntry(Kind export_kind, FlyString export_name_, FlyString local_or_import_name_)
+            : kind(export_kind)
+            , export_name(move(export_name_))
+            , local_or_import_name(move(local_or_import_name_))
+        {
+        }
 
-        // Has just one of ones below
-        FlyString local_or_import_name;
+        bool is_module_request() const
+        {
+            return m_module_request;
+        }
```

Comment:
Same issues as in the previous modules PR - please make sure that getter and member names match :)

Reply:
Here `m_module_request` is implicitly converted to bool, I have added `!= nullptr` to clarify this.

Reply:
Much clearer, thanks!